### PR TITLE
Update README.md to include info on all TOR0 rules

### PR DESCRIPTION
**Pull request by kit1980**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,18 @@ To get the LU factorization see `torch.lu`, which can be used with `torch.lu_sol
 
 `X = torch.solve(B, A).solution` should be replaced with `X = torch.linalg.solve(A, B)`.
 
+### TOR002 Likely typo `require_grad` in assignment. Did you mean `requires_grad`?
+
+This is a common misspelling that can lead to silent performance issues.
+
+### TOR003 Please pass `use_reentrant` explicitly to `checkpoint`
+
+The default value of the `use_reentrant parameter` in `torch.utils.checkpoint` is being changed
```
Comment (clee2000) [inline README.md:76]: Should it be`use_reentrant` parameter instead of `use_reentrant parameter`?

File: README.md [hunk 2]
```diff
@@ -67,6 +67,18 @@ To get the LU factorization see `torch.lu`, which can be used with `torch.lu_sol
...
+from `True` to `False`. In the meantime, the value needs to be passed explictly.
```
Comment (clee2000) [inline README.md:77]: explictly -> explicitly
(missing an i after the c)

**Review by clee2000 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (malfet) [inline README.md:77]:
```suggestion
from `True` to `False`. In the meantime, the value needs to be passed explicitly.
```

File: README.md (hunk 1 above)
Comment (malfet) [inline README.md:76]:
```suggestion
The default value of the `use_reentrant` parameter in `torch.utils.checkpoint` is being changed
```

**Review by malfet — APPROVED** [review]

**Pull request merged**